src/Microsoft.AspNetCore.Razor.Evolution/RazorEngineBuilderExtensions.cs

```diff
@@ -26,6 +27,24 @@ public static IRazorEngineBuilder AddDirective(this IRazorEngineBuilder builder,
             return builder;
         }
 
+        public static IRazorEngineBuilder AddTargetExtension(this IRazorEngineBuilder builder, IRuntimeTargetExtension extension)
```

Comment:
Let's quickly come up with a better name, or we can file an issue to revisit. I want to get this done to unblock use of it in MVC.